module/nilai.go

```diff
@@ -0,0 +1,54 @@
+package module
+
+import (
+	"context"
+
+	nilai "github.com/tokopedia/code-review-workshop/core/nilai"
+	"github.com/tokopedia/tokopoints/errors"
+	"github.com/tokopedia/tokopoints/tracer"
+)
+
+func GetNilaiSiswa(ctx context.Context, nim int64, sekolah, kelas string, mapel_id int64) (resp GetNilaiSiswaResponse, err error) {
+	span, ctx := tracer.StartSpanFromContext(ctx)
+	defer span.Finish()
+
+	if nim != 0 && mapel_id != 0 {
+		nilai, err := nilai.GetNilaiSiswaByID(ctx, nim, mapel_id)
+		if err != nil {
+			return resp, errors.AddTrace(err)
+		}
+
+		detailSekolah, err := nilai.GetDetailSekolah(ctx, nilai.Sekolah[0])
```

Comment:
kasih pengecekan dulu kalo `len(nilai.Sekolah) > 0` baru ambil detail sekolah biar gak index out of bound